molecule/generic_files/playbook.yml

```diff
@@ -2,4 +2,4 @@
 - name: Converge
   hosts: all
   roles:
-    - role: borgbackup
+    - ../../..
```

Comment:
Really, really weird. Never had to do this in any role/scenario ever ...

Reply:
The problem here is, that if a specific string is used, the folder needs to be named the same.
So for testing like this, to keep the environment independent, this uses a relative path to the root directory.

I totally agree that it looks weird though, but it's more "bulletproof"

Reply:
Let me check if there's a better way...

Reply:
Not sure how it works, but Jeff Geerling uses the full role name as it is on Galaxy, but that also doesn't work for me :confused: 

example: https://github.com/geerlingguy/ansible-role-nginx/blob/master/molecule/default/converge.yml

Reply:
@dverhelst Aah, found out why, it's because he moves the repo into a folder by that name in Travis: https://github.com/geerlingguy/ansible-role-nginx/blob/master/.travis.yml#L18

So really not sure there is another way than setting the relative path to the root folder...

Reply:
I've reverted this for now, as it's only an issue for local development.
Will just change this manually when I run tests.